grpc/tests/grpctest.cpp

```diff
@@ -46,8 +46,26 @@ class ServiceImpl final : public MyGame::Example::MonsterStorage::Service {
                                   const flatbuffers::BufferRef<Stat> *request,
                                    ::grpc::ServerWriter< flatbuffers::BufferRef<Monster>>* writer)
                                   override {
-    assert(false);  // We're not actually using this RPC.
-    return grpc::Status::CANCELLED;
+       fbb_.Clear();
+       std::cout << "Hello, " << request->GetRoot()->id()->str()
```

Comment:
Don't think we need output on the server. Just use the `id()` in the generation of a response.